torchx/test/torchx/nx_doctest_test.exs

```diff
@@ -13,33 +13,31 @@ defmodule Torchx.NxDoctestTest do
       |> Kernel.++(
         to_binary: 2,
         default_backend: 2,
-        add: 2,
         round: 1,
```

Comment:
Let's add documentation to why we are not including some doctests. For example

  # Those are not callbacks
  default_backend: 2,

  # We require explicit transfer for those
  to_binary: 2,
  to_scalar: 1,

  # We round differently than Nx for now
  round: 1,

```

and so on!